cdf-core/cdf/js-modules/components/filter/extensions/renderers.js

```diff
@@ -61,7 +61,7 @@ define([
         notificationSelectionLimit: function($tgt, model, configuration) {
           var footer, viewModel;
           viewModel = $.extend(true, model.toJSON(), configuration);
-          footer = Mustache.render("{{#reachedSelectionLimit}}\n<div class=\"filter-root-notification\">\n  <div class=\"filter-root-notification-icon\" />\n  <div class=\"filter-root-notification-text\">\n    The selection limit\n    (<span class=\"filter-notification-highlight\">{{Root.options.selectionStrategy.limit}}</span>)\n    for specific items has been reached.\n  </div>\n</div>\n{{/reachedSelectionLimit}}", viewModel);
+          footer = Mustache.render(BaseFilter.templates["Root-notification-selection-limit"], viewModel);
           return $tgt.find('.filter-root-footer').html(footer);
         }
       });
```

Comment:
I am not sure that the strings should be replaced by the predefined template.
In fact, I am not sure that this file should exist at all. 
AFAIK, this file is invoked by some addIns, so the code could be moved there and this file could be removed.